pkg/apis/servicecatalog/validation/instance.go

```diff
@@ -88,7 +89,9 @@ func validateServiceInstanceSpec(spec *sc.ServiceInstanceSpec, fldPath *field.Pa
 	allErrs := field.ErrorList{}
 
 	allErrs = append(allErrs, validateObjectReferences(spec, fldPath)...)
-	allErrs = append(allErrs, validatePlanReference(&spec.PlanReference, fldPath)...)
+	if !spec.UserProvided {
+		allErrs = append(allErrs, validatePlanReference(&spec.PlanReference, fldPath)...)
+	}
```

Comment:
Should this check to make sure that Plan is blank in the case of a UPS Instance?